.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,29 @@
+name: Github Pages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
```

Comment:
why not `ubuntu-latest`?

Reply:
I definitely did not copy paste this from their example :clown_face: